private/pkg/manifest/module.go

```diff
@@ -31,13 +31,13 @@ var (
 	digestTypeToHashKind = map[DigestType]modulev1alpha1.HashKind{
 		DigestTypeShake256: modulev1alpha1.HashKind_HASH_KIND_SHAKE256,
 	}
+	blockSize = 4098
```

Comment:
You can move this const to be within the func. [If it's a local usage it's ok to declare it in a smaller scope](https://www.notion.so/bufbuild/Go-Code-Standards-ff40382e6b104c1097a017f9565d54db#17b252823cd5404b83a50b82ed295c3d).